### PR TITLE
Add pibooth_startup and pibooth_cleanup hooks

### PR DESCRIPTION
**Pull request by werdeil**
First work for #127, @anxuae tell me what do you think

### CODE REVIEW

**Comment by anxuae** [comment]
Nice job guy!

File: pibooth/booth.py [hunk 1]
```diff
@@ -328,13 +325,8 @@ def main_loop(self):
 
         finally:
             self.makers_pool.quit()
-            self.led_startup.quit()
-            self.led_preview.quit()
-            self.led_capture.quit()
-            self.led_print.quit()
+            self._plugin_manager.hook.pibooth_cleanup(app=self)
```
Comment (anxuae) [inline pibooth/booth.py:328]: In the hookspec, you have defined the config and window can be used.
Thus you have to pass kwargs: cfg=self._config and win=self._window

File: pibooth/booth.py [hunk 2]
```diff
@@ -286,8 +283,8 @@ def main_loop(self):
         """
         try:
             clock = pygame.time.Clock()
-            self.led_startup.switch_on()
             self._initialize()
+            self._plugin_manager.hook.pibooth_startup(app=self)
```
Comment (anxuae) [inline pibooth/booth.py:287]: In the hookspec, you have defined the config and window can be used.
Thus you have to pass kwargs: cfg=self._config and win=self._window

File: pibooth/plugins/hookspecs.py [hunk 3]
```diff
@@ -4,6 +4,18 @@
 
 hookspec = pluggy.HookspecMarker('pibooth')
 
+#--- Pibooth startup and cleanup ----------------------------------------------
+
+@hookspec
+def pibooth_startup(cfg, app, win):
+    """Actions performed at the startup of pybooth
```
Comment (anxuae) [inline pibooth/plugins/hookspecs.py:11]: pibooth (or you want to rename the project ;-) ;-) ) 
add a point

File: pibooth/plugins/hookspecs.py [hunk 4]
```diff
@@ -4,6 +4,18 @@
...
+    """
+
+@hookspec
+def pibooth_cleanup(cfg, app, win):
+    """Actions performed at the cleanup of pybooth
```
Comment (anxuae) [inline pibooth/plugins/hookspecs.py:16]: pibooth (or you want to rename the project ;-) ;-) ) 
add a point

File: pibooth/booth.py (hunk 1 above)
Comment (werdeil) [reply pibooth/booth.py:328]: I removed them from the hook spec for now.

File: pibooth/plugins/hookspecs.py (hunk 3 above)
Comment (werdeil) [reply pibooth/plugins/hookspecs.py:11]: Corrected

File: pibooth/plugins/hookspecs.py (hunk 4 above)
Comment (werdeil) [reply pibooth/plugins/hookspecs.py:16]: Corrected

**Pull request merged**